package.json

```diff
@@ -117,6 +117,7 @@
     "ts:check": "tsc --noEmit && tsc -p tsconfig.examples.json --noEmit",
     "start": "tsx --tsconfig tsconfig.examples.json",
     "start:bee": "yarn start -- examples/agents/bee.ts",
+    "start:bee_telemetry": "INSTRUMENTATION_ENABLED=true node --enable-source-maps --experimental-loader=@opentelemetry/instrumentation/hook.mjs --import ./examples/helpers/telemetry.js ./examples/agents/bee_instrumentation.js",
```

Comment:
I think that `tsx` can be start with a custom loader

Reply:
Unfortunately, this doesn't work. When I update the script to 
```
ODE_OPTIONS=\"--experimental-loader=@opentelemetry/instrumentation/hook.mjs\" tsx --import ./examples/helpers/telemetry.ts ./examples/agents/bee_instrumentation.ts
```

I get the error:
```
 Error: 'import-in-the-middle' failed to wrap 'xxx/examples/agents/bee_instrumentation.ts'
 ```
 
 The alternative solution:
 ```
 // bootstrap.ts
import './helpers/telemetry.ts';
import './agents/bee_instrumentation.ts';
```
does not work either. It must be imported via the custom loader.